### PR TITLE
update travis.yum

### PR DESCRIPTION
**Pull request by djoudi**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,6 +26,10 @@ matrix:
       env: COMPOSER_OPTIONS="--prefer-lowest --prefer-stable"
     - php: 7.0
       env: xdebug="yes" COMPOSER_OPTIONS=""
+      php: 7.1
+      env: xdebug="yes" COMPOSER_OPTIONS=""
+      php: 7.2
+      env: xdebug="yes" COMPOSER_OPTIONS=""
```
Comment (xabbuh) [inline .travis.yml]: I would not add these to builds. Just using 7.1 and 7.2 without Xdebug is enough.

File: .travis.yml [hunk 2]
```diff
@@ -14,6 +14,9 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```
Comment (xabbuh) [inline .travis.yml]: PHP 7.0 is already covered below

**Review by xabbuh — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -11,17 +11,18 @@ env:
     - COMPOSER_OPTIONS="--prefer-stable"
 
 php:
-  - 5.4
```
Comment (xabbuh) [inline .travis.yml:14]: Please keep the existing build jobs for now. I will consider which PHP versions I want to support during the next days. But as long as the Composer constraints do not limit the usable PHP versions I do not want to not test them anymore.

File: composer.json [hunk 4]
```diff
@@ -15,7 +15,7 @@
         "php": ">=5.3.0",
         "php-xapi/exception": "^0.2.0",
         "php-xapi/json-test-fixtures": "^2.0",
-        "php-xapi/model": "^1.1.1 || ^2.0.1",
+        "php-xapi/model": "^1.0 || ^2.0 || 3.0",
```
Comment (xabbuh) [inline composer.json]: This line should be `"php-xapi/model": "^1.1.1 || ^2.0.1 || 3.0",`.

**Review by xabbuh — CHANGES_REQUESTED** [review]

**Comment by djoudi** [comment]
Fatal error: Cannot use 'Object' as class name as it is reserved in /home/travis/build/php-xapi/serializer/vendor/php-xapi/model/src/Object.php on line 19

**Comment by xabbuh** [comment]
Thank you for this PR @djoudi. I have fixed this together with #41 after making some releases in other packages in #43.

**Pull request closed without merging**